pkg/controller/import-controller.go

```diff
@@ -509,7 +509,7 @@ func (r *ImportReconciler) createImportEnvVar(pvc *corev1.PersistentVolumeClaim)
 		if err != nil {
 			return nil, err
 		}
-		podEnvVar.insecureTLS, err = r.isInsecureTLS(pvc)
+		podEnvVar.insecureTLS, err = r.isInsecureTLS(pvc, cdiConfig)
```

Comment:
Just to complete the comment from https://github.com/kubevirt/containerized-data-importer/pull/1754/files#r617498355, I think we should change this code to read the `CDI` CR instead of the `CDIConfig` CR.

Reply:
As Michael said the CDIConfig is a Readonly copy and readable by everyone, vs not everyone would have permission to read the CDI CR. That is why we have the duplication.